client/app/(auth)/login.tsx

```diff
@@ -102,14 +95,24 @@ const Login = () => {
     }
   };
 
-  const handleLogin = async (data: LoginFormData) => {
+  const handleLogin = async (model: LoginFormModel) => {
     setIsLoading(true);
     try {
       await signInWithEmailAndPassword(
         auth,
-        data.email.trim().toLowerCase(),
-        data.password,
+        model.email.trim().toLowerCase(),
+        model.password,
       );
+
+      if (!isUserRegistered) {
+        router.replace("/(auth)/createProfile");
+      }
+
+      if (!isTeamRegistered) {
+        router.replace("/(auth)/createTeam");
+      }
+
+      router.replace("/(protected)");
```

Comment:
We don't need these `router.replace()` calls as this should be handled at the _layout.tsx level